src/Theme_Command.php

```diff
@@ -453,6 +464,11 @@ protected function get_item_list() {
 				'author' => $theme->get('Author'),
 			);
 
+			// Compare version and update information in theme list.
+			if ( isset( $theme_version_info[ $key ] ) && false === $theme_version_info[ $key ] ) {
+				$items[ $file ]['update'] = 'This version is higher than expected!';
```

Comment:
Toning down the message here as well.
```suggestion
				$items[ $file ]['update'] = 'version higher than expected';
```